charts/dex/values.yaml

```diff
@@ -328,3 +328,21 @@ networkPolicy:
   #    ports:
   #      - port: 636
   #        protocol: TCP
+
+# -- List of extra manifests to deploy. Will be passed through `tpl` to support templating
+extraObjects: []
+  # - apiVersion: external-secrets.io/v1beta1
+  #   kind: ExternalSecret
+  #   metadata:
+  #     name: my-external-secret
+  #     namespace: '{{ .Release.Namespace }}'
```

Comment:
Out of curiosity: is the namespace really necessary? Wouldn't helm take care of that? If it's not necessary, I'd probably leave it out of the example as well.

Reply:
I just wanted to highlight that due to usage of the `tpl` function it is possible to use helm internal functions.

Reply:
How about `name: {{ .Release.Name }}` or something like that instead?